gui/default/assets/lang/lang-en.json

```diff
@@ -47,12 +47,14 @@
    "Copyright © 2014-2017 the following Contributors:": "Copyright © 2014-2017 the following Contributors:",
    "Creating ignore patterns, overwriting an existing file at {%path%}.": "Creating ignore patterns, overwriting an existing file at {{path}}.",
    "Danger!": "Danger!",
+   "Data is loading...": "Data is loading...",
    "Deleted": "Deleted",
    "Device": "Device",
    "Device \"{%name%}\" ({%device%} at {%address%}) wants to connect. Add new device?": "Device \"{{name}}\" ({{device}} at {{address}}) wants to connect. Add new device?",
    "Device ID": "Device ID",
    "Device Identification": "Device Identification",
    "Device Name": "Device Name",
+   "Device which has done the latest modification": "Device which has done the latest modification",
```

Comment:
"Device that made the last modification"?

Reply:
"Device on which the last modification was done"? Or closer to yours "Device where the last modification originated?"  
And "originated"/"was done"/"has happened" could be used interchangeably. Originated is the most elegant, but as you say maybe technical. Has happened sounds so passive (it wasn't happenstance but the user - hopefully). So was done seems most appropriate, but also ugly.